pennylane/qnode_new/device_jacobian.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Device Jacobian QNode.
+
+A QNode that delegates all gradient computations directly to the device.
+"""
+from pennylane.qnode_new.jacobian import JacobianQNode
+
+
+class DeviceJacobianQNode(JacobianQNode):

```

Comment:
If this class would not be a device in itself (and inherit from any device), maybe having ``Device`` in its name would not be the best choice. One that pops in my mind based on the docstring could be ``DelegatedJacobianQNode``, but there is for sure room for improvement there!

Reply:
This is true. Something else I was thinking was `CustomGradientQNode`, but even this is not entirely correct.

Reply:
@co9olguy, and preferences for a name?

Reply:
I think all names are fine. Device in the name is not too misleading as "Device" is not at the end.